src/Implement/MainManager.cpp

```diff
@@ -0,0 +1,80 @@
+#include "MainManager.h"
+
+MainManager *MainManager::only_mainManager = nullptr;
+
+void SigHandle(int sig, siginfo_t *info, void *ctx) {
```

Comment:
Please move this into an anonymous namespace to avoid interfering with other .cpp files.

Reply:
Done